src/app/views/sidebar/resource-explorer/panels/PathsReview.tsx

```diff
@@ -21,37 +38,126 @@ export interface IPathsReview {
 
 const PathsReview = (props: IPathsReview) => {
   const dispatch: AppDispatch = useDispatch();
-  const { resources: { paths: items } } = useAppSelector(
-    (state) => state
-  );
+  const items = useAppSelector((state) => state.resources.paths);
   const { isOpen } = props;
-  const headerText = translateMessage('Selected Resources') + ' ' + translateMessage('Preview');
+  const headerText =
+    translateMessage('Selected Resources') + ' ' + translateMessage('Preview');
   const [selectedItems, setSelectedItems] = useState<IResourceLink[]>([]);
+  const [
+    showVersionErrorMessage,
+    {
+      toggle: toggleShowVersionErrorMessage,
+      setTrue: setShowVersionErrorMessageTrue,
+      setFalse: setShowVersionErrorMessageFalse
+    }
+  ] = useBoolean(false);
 
   const columns = [
-    { key: 'url', name: 'URL', fieldName: 'url', minWidth: 300, maxWidth: 350, isResizable: true }
+    {
+      key: 'url',
+      name: 'URL',
+      fieldName: 'url',
+      minWidth: 300,
+      maxWidth: 350,
+      isResizable: true
+    }
   ];
 
   const generateCollection = () => {
     const content = generatePostmanCollection(items);
     const filename = `${content.info.name}-${content.info._postman_id}.postman_collection.json`;
     downloadToLocal(content, filename);
-  }
-
+  };
+  const getDescriptionUrl = () => {
+    if (items.length > 0 && items[0].version === 'beta') {
+      return GRAPH_BETA_DESCRIPTION_URL;
+    }
+    return GRAPH_V1_DESCRIPTION_URL;
+  };
+  const getVersionsCount = () => {
+    const versions = items.map((item) => item.version);
```

Comment:
we should also add another check here for the case where items is defined and return an empty Set

Reply:
thanks fixed